configs/YYRC-EAGLEF405V0

```diff
@@ -0,0 +1,116 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F405
+
+#define BOARD_NAME        SPEDIXF405
+#define MANUFACTURER_ID   SPDX
```

Comment:
- this is not supposed to be here.
- is this a SPEDIXF405 clone?

Reply:
No,it is not aSPEDIXF405 clone.because，The config pull request is change. Use it as a reference.Thank you!  you tell me the problem, I will modify it to meet the requirements.

Reply:
No,it is not aSPEDIXF405 clone.because，The config pull request is change. Use it as a reference.Thank you!  you tell me the problem, I will modify it to meet the requirements.